tests/scaffolders/manager.py

```diff
@@ -60,7 +60,11 @@ class ScaffolderManagerTest(unittest.TestCase):
 
   @classmethod
   def setUpClass(cls):
-    """Setup the tests by registering three scaffolders."""
+    """Setup tests by registering three scaffolders and removing all others."""
+    scaffolders = [x for _, x in manager.ScaffolderManager.GetScaffolders()]
```

Comment:
Please move this functionality to the manager, rather than use a list comprehension like this.

Reply:
this is not a functionality I would like to have in the manager, except if you just want "ClearAllScaffolders" function or something equivalent in the manager, this is just to make sure there aren't any scaffolders registered in the manager before starting the tests..... 

Do you want me to add a function to the manager to just clear everything out?